awxkit/awxkit/cli/options.py

```diff
@@ -124,6 +126,13 @@ def jsonstr(v):
                 if param['type'] == 'id' and not kwargs.get('help'):
                     kwargs['help'] = 'the ID of the associated  {}'.format(k)
 
+                if param['type'] == 'json':
+                    help_parts = []
+                    if kwargs.get('help'):
+                        help_parts.append(kwargs['help'])
+                    help_parts.append('You can also specify a file path e.g., @path/to/file.yml')
+                    kwargs['help'] += ' '.join(help_parts)
```

Comment:
I don't understand how you are doing both `+=` _and_ the `.join`. Since you're already joining the existing help content (which is a good pattern to use), then it should be `kwargs['help'] = ' '.join(help_parts)`

Reply:
Yep, that's a typo.